home/export_content_pages.py

```diff
@@ -208,6 +210,11 @@ def _export_content_page(self, page: ContentPage, structure: str) -> None:
          * We should use the parent slug (which is expected to be unique per
            locale (probably?)) instead of the parent title.
         """
+        export_lang_code = ""
+        for lang_code, lang_dn in get_content_languages().items():
+            if lang_dn == str(page.locale):
+                export_lang_code = lang_code
```

Comment:
Can we not just do `page.locale.language_code` to get the language code for the current page?